apps/projects/app/components/Shared/FilterBar/FilterDropDown.js

```diff
@@ -7,130 +7,117 @@ import { springs, GU, theme } from '@aragon/ui'
 }
-FilterDropDown.defaultProps = {
-  type: 'filter',
-  caption: '',
+OverflowDropDown.defaultProps = {
   enabled: true,
 }
 
-const borders = {
-  'overflowTop': '3px 3px 0 0',
-  'overflowMiddle': '0',
-  'overflowBottom': '0 0 3px 3px',
-  'filter': '3px',
-}
+export const FilterDropDown = ({ caption, type, children, enabled }) => {
+  const [ opened, setOpened ] = useState(false)
+  const handleClickOut = () => setOpened(false)
+  const handleBaseButtonClick = () => {
+    if (enabled) setOpened(!opened)
+  }
 
-const DropDownButtonFilter = ({ disabled, handleBaseButtonClick, caption, openProgress, handleClose, type, children }) => (
-  <Main>
-    <FilterButton
-      onClick={handleBaseButtonClick}
-      disabled={disabled}
-      width="128px"
-      style={{ borderRadius: borders[type] }}
-    >
-      <div css={`display: flex; width: 100%; justify-content: space-between; padding: 0 ${2 * GU}px`}>
-        {caption}
-        <animated.div
-          style={{
-            height: '16px',
-            transformOrigin: '50% 70%',
-            transform: openProgress.interpolate(
-              t => `rotate(${t * 180}deg)`
-            ),
-          }}
-        >
-          <IconDropArrow color={`${theme.surfaceContentSecondary}`} />
-        </animated.div>
-      </div>
-    </FilterButton>
+  return (
+    <SpringWrap handleClickOut={handleClickOut} opened={opened}>
+      {(openProgress) => (
+        <Main>
+          <FilterButton
+            onClick={handleBaseButtonClick}
+            disabled={!enabled}
+            width="128px"
+            style={{ borderRadius: borders[type] }}
+          >
+            <div css={`display: flex; width: 100%; justify-content: space-between; padding: 0 ${2 * GU}px`}>
+              {caption}
+              <animated.div
+                style={{
+                  height: '16px',
+                  transformOrigin: '50% 70%',
+                  transform: openProgress.interpolate(
+                    t => `rotate(${t * 180}deg)`
+                  ),
+                }}
+              >
+                <IconDropArrow color={`${theme.surfaceContentSecondary}`} />
+              </animated.div>
+            </div>
+          </FilterButton>
 
-    <Popup onClick={handleClose} style={{ opacity: openProgress }}>
-      {children}
-    </Popup>
-  </Main>
-)
-DropDownButtonFilter.propTypes = {
-  disabled: PropTypes.bool.isRequired,
-  handleBaseButtonClick: PropTypes.func.isRequired,
+          {opened &&
+            <Popup style={{ opacity: openProgress }}>
+              {children}
+            </Popup>
+          }
+        </Main>
+      )}
+    </SpringWrap>
+  )
+}
+FilterDropDown.propTypes = {
   caption: PropTypes.string.isRequired,
-  openProgress: PropTypes.object.isRequired,
-  handleClose: PropTypes.func.isRequired,
-  type: PropTypes.string.isRequired,
   children: PropTypes.node.isRequired,
+  enabled: PropTypes.bool.isRequired,
+  type: PropTypes.string.isRequired,
```

Comment:
It no longer uses `type`, right?

Reply:
Indeed. It could be in this PR, but next makes it.. different. Next PR has dynamic Overflow.

Reply:
Removed `type` for now.